src/containers/Guide/GuideResyncing/index.tsx

```diff
@@ -2,53 +2,152 @@ import React, {FC, ReactNode} from 'react';
 
 import {DocContainer, DocImage, DocList, DocSubSection, TableBorderGrid} from 'components';
 
-import PrimaryValidatorUpgradeRequest from './PrimaryValidatorUpgradeRequest.png';
+import ResyncPrimaryValidatorUpdated from './ResyncPrimaryValidatorUpdated.png';
+import ResyncTriggerNetworkError from './ResyncTriggerNetworkError.png';
+import ResyncTriggers from './ResyncTriggers.png';
+import ResyncUpgradeNotice from './ResyncUpgradeNotice.png';
+import ResyncUpgradeRequest from './ResyncUpgradeRequest.png';
 
 const GuideResyncing: FC = () => {
-  const renderPrimaryValidatorUpdateNotice = (): ReactNode => (
+  const renderOverview = (): ReactNode => (
     <>
-      <p>
-        This notice is sent from banks to confirmation validators to indicate that the requesting bank is leaving the
-        current network and switching to a new primary validator (a new network).
-      </p>
-      <p>Banks will always set their primary validator to the validator that is the:</p>
-
+      <p>Banks will always set their primary validator to the validator that is:</p>
       <DocList variant="ul">
-        <li>most trusted</li>
+        <li>the most trusted</li>
         <li>online</li>
         <li>configured as a primary validator</li>
       </DocList>
 
+      <p>Confirmation validators will always:</p>
+      <DocList variant="ul">
+        <li>follow the guidance of their most trusted bank</li>
+      </DocList>
+    </>
+  );
+
+  const renderPrimaryValidatorUpdated = (): ReactNode => (
+    <>
       <p>
-        When a bank changes primary validators, any confirmation validators who have that bank set as their most trusted
-        need to be made aware of the change in order to correctly sync to the new primary validator.
+        The last step in the resync process is the notice that is sent from banks to their CV's to indicate that they
+        are leaving the current network and switching to a new PV (a new network).
       </p>
+
+      <DocImage alt="primary validator updated" maxWidth={580} src={ResyncPrimaryValidatorUpdated} />
+
       <p>
         Any confirmation validators that receive this request who do not have the requesting bank set as their most
         trusted will delete the requesting bank. This is because the bank is indicating that it is leaving the network,
-        and therefore a connection is no longer needed.
+        and therefore a connection is no longer needed. Any confirmation validators that do have the requesting bank set
+        as their most trusted bank will follow the guidance of that bank and sync to the new primary validator.
       </p>
     </>
   );
 
-  const renderUpgradeRequest = (): ReactNode => (
+  const renderResyncProcess = (): ReactNode => (
+    <>
+      <h1 className="DocContainer__page-title GuideIntroduction__page-title">Resync Process</h1>
```

Comment:
:( I was trying to avoid this type of classes, because whoever changes the styling in either of the two classes will almost certainly have no idea it is also effecting this one here. Do you think having more than one `h1` is something we should accodomate for in our docs?

Reply:
I'll move this it's own page.